keel-core/src/test/kotlin/com/netflix/spinnaker/keel/events/ResourceStatusTests.kt

```diff
@@ -50,7 +50,7 @@ internal class ResourceStatusTests : JUnit5Minutests {
     val repository = combinedInMemoryRepository(clock)
```

Comment:
should this just change to a `ResourceRepository`, or is the combined repository used in the rest of the tests?